src/tasks/bundler.rs

```diff
@@ -1,58 +1,86 @@
 //! Bundler service responsible for managing bundles.
+use std::sync::Arc;
```

Comment:
style: 
remove newlines between uses
use a full `crate::` path instead of `super::`